config/samples/resources/alloydbcluster/regular-cluster/alloydb_v1beta1_alloydbcluster.yaml

```diff
@@ -44,6 +44,14 @@ spec:
   encryptionConfig:
     kmsKeyNameRef: 
       name: alloydbcluster-dep-regular
+  maintenanceUpdatePolicy:
+    maintenanceWindows:
+      - day: WEDNESDAY
```

Comment:
Hi @vmiglani, have you verified that the maintenanceUpdatePolicy are indeed configured in the resource? I was double checking because the way how the list is indented may or may not work properly and I want to ensure the sample is working as expected.